seqr/views/apis/staff_api.py

```diff
@@ -404,7 +409,7 @@ def _parse_anvil_metadata(project, individual_samples, get_saved_variants_by_fam
                     'Alt': variant['alt'],
                     'hgvsc': (variant['main_transcript'].get('hgvsc') or '').split(':')[-1],
                     'hgvsp': (variant['main_transcript'].get('hgvsp') or '').split(':')[-1],
```

Comment:
no its slightly different, if you have `dict = {'hgvsc': None}` then what I have here returns the empty string but with that change it would return None